toolkit/tools/pkg/imagecustomizerlib/testdata/runscripts-config.yaml

```diff
@@ -0,0 +1,5 @@
+PostInstallScripts:
```

Comment:
Is this missing from `customizeutils_test.go`? Looks like `addfiles-config.yaml` is getting used.

Reply:
No. The install scripts need a full image to be able to run. So, it is not really practical to add tests for them to the current set of UTs. This file exists just to assist with manual testing.

At some point, we should probably add a set of tier 2 tests that can have more requirements and allows to run for longer.